care/emr/management/commands/load_fixtures.py

```
@@ -11,6 +11,8 @@
 from faker import Faker
 from faker.providers.geo import Provider as GeoProvider
 
+# Override the validate_valueset function to skip validation for fixtures
+import care.emr.utils.valueset_coding_type  # noqa  # isort:skip
```

Ruff (0.13.3)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)